src/ir/eh-utils.h

```diff
@@ -20,6 +20,8 @@
 namespace wasm {
 
 class Expression;
+class Function;
+class Module;
```

Comment:
I don't think I noticed this before, but looks like it avoids including wasm.h by adding these forward declarations? Given there are 3 of them I think a single line that includes "wasm.h" would be shorter.

Reply:
Done. I tried to avoid includes if possible (just as general recommendations), but in this case that might not matter much anyway...